### PR TITLE
Submission

### PR DESCRIPTION
**Pull request by carriewright11**
Starting to add more detail about the submission process and finding the correct links for people to see more information

- https://grants.nih.gov/grants/how-to-apply-application-guide/forms-h/general-forms-h.pdf
- https://grants.nih.gov/grants/how-to-apply-application-guide/forms-h/career-forms-h.pdf

I think these links are the ones we want to read through more for additional info

### CODE REVIEW

File: 05-submission.Rmd [hunk 1]
```diff
@@ -6,35 +6,41 @@ ottrpal::set_knitr_image_path()
...
+  - Check with the Intramural Research Program
+- Other funding agreements
+  - Plans should be submitted prior to the release of funds
 
-this is from the website:
+## Other Submission Tips:
 
-The DMS Plan should be submitted as follows:
+- Resource sharing plans will not be going away (but will **not** be about data and will instead focus on tools and model organisms)
+- There will not be a separate [genomic data sharing](https://sharing.nih.gov/genomic-data-sharing-policy/about-genomic-data-sharing/gds-policy-overview) plan, plans to comply with this policy should simply be included in the DMS plan
```
Comment (avahoffman) [inline 05-submission.Rmd:31]:
```suggestion
- There will not be a separate [genomic data sharing](https://sharing.nih.gov/genomic-data-sharing-policy/about-genomic-data-sharing/gds-policy-overview) plan. Genomic data sharing plans should simply be included in the overall DMS plan.
```

File: 05-submission.Rmd [hunk 2]
```diff
@@ -6,35 +6,41 @@ ottrpal::set_knitr_image_path()
...
-- can update plan if not accepted (just in time)
-- can also update during regular reporting intervals - thus need to show that you made a plan but it's OK if things change (but these will need to be approved)
-    
- 11. Other details
- -Unique persistent identifiers are not required (such as a DOI but they are encouraged)
-  
-  Investigators are expected to report progress made on data sharing in their Research Progress Performance Report (RPPR). 
+- You can update your DMS plan if is not accepted during the [Just-in-Time](https://grants.nih.gov/grants/policy/nihgps/HTML5/section_2/2.5.1_just-in-time_procedures.htm) process
+- You can also update your DMS plan during regular reporting intervals (but these changes will need to be approved)
+- Investigators are expected to report progress made on data sharing in their Research Progress Performance Report (RPPR)
```
Comment (avahoffman) [inline 05-submission.Rmd:43]: `sessionInfo()` appears at the bottom of this page. Can we move it to the appendix somewhere?

**Review by avahoffman — APPROVED** [review]
LGTM!

File: 05-submission.Rmd (hunk 2 above)
Comment (carriewright11) [reply 05-submission.Rmd:43]: yes! thanks for noticing that I think we already have it in the appendix

File: 05-submission.Rmd [hunk 3]
```diff
@@ -6,42 +6,41 @@ ottrpal::set_knitr_image_path()
...
- Implementation details
-- other plans field added to forms H filed of the PHS 398 research plan form/career development supplemental form
-- Resource sharing plans will not be going away (but will not be about data - tools and model organisms)- instructions will be updated
-- No longer separate data sharing or genomic data sharing plans
+# Presubmission Tips
 
 
+- Extramural (grants)
+    - DMS plans should be included within the “Other Plan(s)" field on the following forms:
+        - for grants not related to career development:[PHS 398 Research Plan](https://grants.nih.gov/grants/how-to-apply-application-guide/forms-h/general-forms-h.pdf).
```
Comment (carriewright11) [inline 05-submission.Rmd:18]:
```suggestion

 - for grants not related to career development: [PHS 398 Research Plan](https://grants.nih.gov/grants/how-to-apply-application-guide/forms-h/general-forms-h.pdf).
 ```

Comment (carriewright11) [reply 05-submission.Rmd:18]: Noticed I was missing a space

**Pull request merged**